.gitignore

```diff
@@ -19,6 +19,14 @@ test/benchmark/baseline.txt
 xcuserdata/
 *.xccheckout
 
+# Allow Visual Studio project files.
+!wren/
+!wren_lib/
```

Comment:
Also, none of the entries in gitignore are top level only. I suppose fixing that would be the preferred way to get rid of these entries.

The CLI project was accidentally removed from the repository when you moved it to another directory because anything with the name `wren` was ignored. Including the `wren` folder containing the CLI project.